src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -329,16 +323,6 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         return super.getItem(NameEncoder.encode(projectName));
     }
 
-    /**
-     * Returns {@code true} if this is a single origin {@link OrganizationFolder}.
-     *
-     * @return {@code true} if this is a single origin {@link OrganizationFolder}.
-     */
-    public boolean isSingleOrigin() {
```

Comment:
This seems to be used here https://github.com/jenkinsci/tuleap-git-branch-source-plugin which has certain activity

Reply:
Mm, OK, I can restore it but deprecate it.